### ISSUE TITLE
[Bug] main.php line 191

### ISSUE THREAD

**Issue by BroCrows** [ItsMax123/StaffMode#7]
**Steps to reproduce the bug**
Idk because it has server happened to me

**Error report**
```
28.07 23:21:50 [Server] Server thread/CRITICAL ErrorException: "Trying to access array offset on value of type null" (EXCEPTION) in "plugins/StaffMode.phar/src/Max/StaffMode/Main" at line 191
28.07 23:21:50 [Server] Server thread/CRITICAL #0 plugins/StaffMode.phar/src/Max/StaffMode/Main(191): pocketmine\utils\Utils::errorExceptionHandler(integer 8, string[51] Trying to access array offset on value of type null, string[103] phar:///plugins/StaffMode.phar/src, integer 191, array[1])
28.07 23:21:50 [Server] Server thread/CRITICAL #1 plugins/StaffMode.phar/src/Max/StaffMode/EventListener(169): Max\StaffMode\Main->exitstaffmode(object pocketmine\Player)
28.07 23:21:50 [Server] Server thread/CRITICAL #2 pmsrc/src/pocketmine/plugin/MethodEventExecutor(42): Max\StaffMode\EventListener->onKick(object pocketmine\event\player\PlayerKickEvent)
28.07 23:21:50 [Server] Server thread/CRITICAL #3 pmsrc/src/pocketmine/plugin/RegisteredListener(80): pocketmine\plugin\MethodEventExecutor->execute(object Max\StaffMode\EventListener, object pocketmine\event\player\PlayerKickEvent)
28.07 23:21:50 [Server] Server thread/CRITICAL #4 pmsrc/src/pocketmine/event/Event(88): pocketmine\plugin\RegisteredListener->callEvent(object pocketmine\event\player\PlayerKickEvent)
28.07 23:21:50 [Server] Server thread/CRITICAL #5 pmsrc/src/pocketmine/Player(3428): pocketmine\event\Event->call()
28.07 23:21:50 [Server] Server thread/CRITICAL #6 pmsrc/src/pocketmine/Player(2012): pocketmine\Player->kick(string[22] Server is white-listed, boolean )
28.07 23:21:50 [Server] Server thread/CRITICAL #7 pmsrc/src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(117): pocketmine\Player->handleLogin(object pocketmine\network\mcpe\protocol\LoginPacket)
28.07 23:21:50 [Server] Server thread/CRITICAL #8 pmsrc/src/pocketmine/network/mcpe/protocol/LoginPacket(145): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleLogin(object pocketmine\network\mcpe\protocol\LoginPacket)
28.07 23:21:50 [Server] Server thread/CRITICAL #9 pmsrc/src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(109): pocketmine\network\mcpe\protocol\LoginPacket->handle(object pocketmine\network\mcpe\PlayerNetworkSessionAdapter)
28.07 23:21:50 [Server] Server thread/CRITICAL #10 pmsrc/src/pocketmine/network/mcpe/protocol/BatchPacket(130): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(object pocketmine\network\mcpe\protocol\LoginPacket)
28.07 23:21:50 [Server] Server thread/CRITICAL #11 pmsrc/src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(109): pocketmine\network\mcpe\protocol\BatchPacket->handle(object pocketmine\network\mcpe\PlayerNetworkSessionAdapter)
28.07 23:21:50 [Server] Server thread/CRITICAL #12 pmsrc/src/pocketmine/Player(3321): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(object pocketmine\network\mcpe\protocol\BatchPacket)
28.07 23:21:50 [Server] Server thread/CRITICAL #13 pmsrc/src/pocketmine/network/mcpe/RakLibInterface(169): pocketmine\Player->handleDataPacket(object pocketmine\network\mcpe\protocol\BatchPacket)
28.07 23:21:50 [Server] Server thread/CRITICAL #14 pmsrc/vendor/pocketmine/raklib/src/server/ServerHandler(95): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string[20] 107.12.208.234 50417, object raklib\protocol\EncapsulatedPacket, integer 0)
28.07 23:21:50 [Server] Server thread/CRITICAL #15 pmsrc/src/pocketmine/network/mcpe/RakLibInterface(109): raklib\server\ServerHandler->handlePacket()
28.07 23:21:50 [Server] Server thread/CRITICAL #16 pmsrc/src/pocketmine/network/mcpe/RakLibInterface(99): pocketmine\network\mcpe\RakLibInterface->process()
28.07 23:21:50 [Server] Server thread/CRITICAL #17 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(113): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
28.07 23:21:50 [Server] Server thread/CRITICAL #18 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(75): pocketmine\snooze\SleeperHandler->processNotifications()
28.07 23:21:50 [Server] Server thread/CRITICAL #19 pmsrc/src/pocketmine/Server(2155): pocketmine\snooze\SleeperHandler->sleepUntil(double 1627514510.0656)
28.07 23:21:50 [Server] Server thread/CRITICAL #20 pmsrc/src/pocketmine/Server(1992): pocketmine\Server->tickProcessor()
28.07 23:21:50 [Server] Server thread/CRITICAL #21 pmsrc/src/pocketmine/Server(1586): pocketmine\Server->start()
28.07 23:21:50 [Server] Server thread/CRITICAL #22 pmsrc/src/pocketmine/PocketMine(311): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[47] /, string[55] /plugins/)
28.07 23:21:50 [Server] Server thread/CRITICAL #23 pmsrc/src/pocketmine/PocketMine(341): pocketmine\server()
28.07 23:21:50 [Server] Server thread/CRITICAL #24 pmsrc(11): require(string[102] phar:///custom-php7.4.phar/src/poc)
28.07 23:21:50 [Server] Server thread/INFO XxChroma[IP] logged out due to Internal server error
28.07 23:21:50 [Server] RakLibServer thread/NOTICE Blocked IP for 5 seconds
```

**Plugin version, Pocketmine version**
Are you using the latest version of the plugin? (Y/N) Yes
Are you using the latest version of pocketmine? (Y/N) Yes

**Plugins used**
Not needed as it there is no other plugin that I have that kicks players

**Comment by BroCrows** [comment]
I removed NameSpaceRemover and didnt get this bug

**Comment by BroCrows** [comment]
Ok I am confused, it isnt NameSpaceRemover's fault!

**Comment by BroCrows** [comment]
This is the same issue as #5

**Comment by ItsMax123** [comment]
I cant help if you dont tell me how it happens

**Comment by BroCrows** [comment]
I think it happens if you dont have the silent join permission

**Comment by BroCrows** [comment]
And like I said in #8, it happens to non-operators

**Comment by MrNinja008** [comment]
@ItsMax123  Same problem for me it gives the same error if i try to join
Error: Trying to access array offset on value of type null
File: plugins/StaffMode_dev-30.phar/src/Max/StaffMode/Main
Line: 191
Type: ErrorException

THIS CRASH WAS CAUSED BY A PLUGIN
BAD PLUGIN: StaffMode v2.0.1

Code:
[182]             $rtorch = Item::get(Item::LIT_REDSTONE_TORCH, 0, 1);
[183]             $rtorch->setCustomName("§cExit StaffMode");
[184]             $rtorch->setNamedTagEntry(new StringTag("staffmode", "true"));
[185]             $rtorch->setLore(["§rRight click to exit StaffMode."]);
[186]             $player->getInventory()->setItem(8, $rtorch);
[187]         }
[188]     }
[189] 
[190]     public function exitstaffmode(Player $player) {
[191]         if($this->staffmodestatus[$player->getName()]) {
[192]             $player->getInventory()->setContents($this->contents[$player->getName()]);
[193]             $player->teleport($this->position[$player->getName()]);
[194]             $player->setGamemode($this->gamemode[$player->getName()]);
[195]             $this->staffmodestatus[$player->getName()] = False;
[196]             Server::getInstance()->addOnlinePlayer($player);
[197]             $player->sendPopup("§cYou are no longer in staffmode.");
[198] 
[199]             //Fake join message
[200]             if($this->config->get("FakeJoin")){
[201]                 $message = $this->getConfig()->get("FakeJoin-Message");

**Comment by BroCrows** [comment]
Now the bug is at 181

**Comment by ItsMax123** [comment]
Fixed in commit 9da74f2
Download here: https://poggit.pmmp.io/ci/ItsMax123/StaffMode/StaffMode/dev:42
